addons/website/views/snippets/s_timeline.xml

```diff
@@ -2,51 +2,52 @@
 <odoo>
 
 <template name="Timeline" id="s_timeline">
-    <section class="s_timeline pt24 pb48">
-        <div class="container s_timeline_line">
-            <div class="s_timeline_row d-block d-md-flex flex-row" data-name="Row">
-                <div class="s_timeline_date"><b>2019</b></div>
-                <div class="s_timeline_content d-flex">
-                    <div class="s_timeline_card s_card card bg-white w-100" data-name="Card" data-snippet="s_card">
-                        <div class="card-body">
-                            <h5 class="card-title">Your title</h5>
-                            <p class="card-text">A timeline is a graphical representation on which important events are marked.</p>
-                        </div>
-                    </div>
-                    <i class="fa fa-1x fa-child bg-o-color-2 rounded-circle s_timeline_icon"/>
+    <section class="s_timeline pt48 pb48" data-vcss="001">
+        <div class="o_container_small">
+            <div class="row">
+                <div class="col pb24 text-center" data-name="Heading">
+                    <h3>Latest news</h3>
+                    <p class="lead">Highlight your history, showcase growth and key milestones.</p>
                 </div>
-                <div class="s_timeline_content"/>
             </div>
-            <div class="s_timeline_row d-block d-md-flex flex-row" data-name="Row">
-                <div class="s_timeline_date"><b>2018</b></div>
-                <div class="s_timeline_content d-flex">
-                    <div class="s_timeline_card s_card card bg-white w-100" data-name="Card" data-snippet="s_card">
-                        <div class="card-body">
-                            <p class="card-text">You can edit, duplicate...</p>
+            <div class="position-relative">
+                <div class="s_timeline_line position-absolute top-0 bottom-0 d-block border-start border-300"/>
+                <div class="s_timeline_row position-relative d-flex gap-5 align-items-start pb-5" data-name="Year">
```

Comment:
This is the only `s_timeline_row` with a `pb-5` class, let's add it to the others as well ;)

edit: since it's not editable by the user, let's use `pb-4` instead. If they want more they can alway edit the text and add an empty line at the end.